js/ui/camera.js

```diff
@@ -302,7 +303,8 @@ class Camera extends Evented {
      * @param {Object} [options]
      * @param {boolean} [options.linear=false] If `true`, the map transitions using
      *     {@link Map#easeTo}. If `false`, the map transitions using {@link Map#flyTo}. See
-     *     {@link Map#flyTo} for information about the options specific to that animated transition.
+     *     those functions and @{link AnimationOptions} for information about options available.
+     * @param {Function} [options.easing] An easing function for the animated transition. See [AnimationOptions](#AnimationOptions).
      * @param {Function} [options.easing] An easing function for the animated transition.
```

Comment:
`options.easing` is duplicated here.  In fact, for consistency, would it make sense to remove `options.easing` (and `options.offset`) in favor of setting the type of `[options]` to `{AnimationOptions}`?  I realize that, at present, this means more clicks for readers of the docs, but my preference would be to solve that problem somehow in the documentation generation step.